lib/rubocop/cop/style/stderr_puts.rb

```diff
@@ -29,6 +29,9 @@ class StderrPuts < Cop
         def on_send(node)
           return unless stderr_puts?(node)
 
+          arguments = node.arguments
```

Comment:
I'd update the pattern instead. It would make the code more compact.

Reply:
Thanks for the reply @bbatsov! I changed the pattern like you suggested. Could you check again? 😉